AC3.2Unit3MidunitAssessment/AC3.2Unit3MidunitAssessment/Album.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Spotify.swift
+//  AC3.2Unit3MidunitAssessment
+//
+//  Created by Ilmira Estil on 10/28/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+internal struct Album {
+    internal let albumTitle: String
+    internal let albumArrayImages: [[String:String]]
+    internal let albumThumbnailURLString: String = ""
+    internal let albumFullImageURLString: String = ""
+    
+    
+    //String:[String:AnyObject]?
+    static func getAlbum(from data: Data) -> [Album]? {
+        do {
+            let albumJSONData: Any = try JSONSerialization.jsonObject(with: data, options: [])
+            guard let albumCasted: [String:AnyObject] = albumJSONData as? [String:AnyObject],
+                let albumArray: [AnyObject] = albumCasted["albums"] as? [AnyObject] else {return nil}
```

Comment:
guard let albums = data["albums"] as? [String: Any] else { return nil }

- should be a dictionary of [String: Any] and not an array of [AnyObject]